src/newgrf.cpp

```diff
@@ -6191,11 +6191,11 @@ static void FeatureMapSpriteGroup(ByteReader &buf)
 	 * B n-id          bits 0-6: how many IDs this definition applies to
 	 *                 bit 7: if set, this is a wagon override definition (see below)
 	 * E ids           the IDs for which this definition applies
-	 * B num-cid       number of cargo IDs (sprite group IDs) in this definition
+	 * B num-cid       number of cargo types (sprite group IDs) in this definition
 	 *                 can be zero, in that case the def-cid is used always
 	 * B cargo-type    type of this cargo type (e.g. mail=2, wood=7, see below)
-	 * W cid           cargo ID (sprite group ID) for this type of cargo
-	 * W def-cid       default cargo ID (sprite group ID) */
+	 * W cid           cargo type (sprite group ID) for this type of cargo
+	 * W def-cid       default cargo type (sprite group ID) */
```

Comment:
I think some of these cargo ID to cargo type changes are misleading. These are not actually `CargoID` nor `CargoType`, they are just something that the spec called a cargo id. (Sometimes they are cargo related, but in that case they pass through TranslateCargo() which *does* return a CargoID.